src/views/GameSettings/__tests__/LocalPlayerSettings.test.tsx

```
@@ -0,0 +1,728 @@
+}));
+
+vi.mock('@/components/LocalPlayerSetup', () => ({
+  default: ({
+    onComplete,
+    onCancel,
+  }: {
+    onComplete: (players: any[], settings: any) => void;
+    onCancel: () => void;
+  }) => (
+    <div data-testid="local-player-setup">
+      <button onClick={() => onComplete([], {})}>Complete Setup</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)